src/css/tabs/receiver.css

```
@@ -559,6 +589,10 @@
         margin-bottom: 10px;
     }
 
+    select.features.rxMode, select.serialRX {
```

This does not need the `.tab-receiver` to maintain consistence with the other styles? If it is a general `feature` style, maybe is better to go in a common css file. Is this if only for this tab, is better to add the `.tab-receiver` to go with the others styles.
```suggestion
    .tab-receiver select.features.rxMode, .tab-receiver select.serialRX {
```